mflowgen/glb_tile/custom-init/outputs/floorplan.tcl

```diff
@@ -52,4 +52,53 @@ setFlipping s
 
 planDesign
 
+proc snap_to_grid {input granularity} {
+   set new_value [expr ceil($input / $granularity) * $granularity]
+   return $new_value
+}
+
+set horiz_pitch [dbGet top.fPlan.coreSite.size_x]
+set srams [get_cells *sram_array*]
+set sram_width [dbGet [dbGet -p top.insts.name *sram_array* -i 0].cell.size_x]
+set sram_height [dbGet [dbGet -p top.insts.name *sram_array* -i 0].cell.size_y]
+set sram_start_y [expr [dbGet top.fPlan.box_lly] + 50]
```

Comment:
Rather than hard coding an offset from the floorplan edge, it would be cool if we could make it do some math to automatically center the glb tiles within glb_top, regardless of the floorplan size. Not essential for now, but it will make the flow more robust to design and technology changes. I did this to place the SRAM macros within the memory tile: https://github.com/StanfordAHA/garnet/commit/2f02e820c57669fbc70f006331295e3cbb977194#diff-89877f7f45b789cca62117de8112f87a